cspatients/views.py

```diff
@@ -231,17 +231,24 @@ def get(self, request):
         entries = PatientEntry.objects.filter(completion_time__isnull=True)
 
         ids = []
-        data = []
+        patient_data = []
         for count, entry in enumerate(entries, start=1):
-            data.append(
+            patient_data.append(
                 f"{count}) {entry.surname} {entry.operation} {entry.indication} {entry.get_urgency_color()}"
             )
             ids.append(f"{count}={entry.id}")
 
-        return Response(
-            {"patient_list": "\n".join(data), "patient_ids": "|".join(ids)},
-            status=status.HTTP_200_OK,
-        )
+        data = {"patient_ids": "|".join(ids)}
+
+        list_number = 0
+        while len(patient_data) > 0:
```

Comment:
This `while` loop makes me nervous 😅 I think it's okay though. I do think that we should move the list size to be defined in a single place though. We could make it configurable in settings, with a default of 10? Or alternatively, just declare it as a constant at the top of the file?

Reply:
Thanks, I moved the list size to settings.

I think the `while` loop will be fine for now. We'll probably have to come up with something different if the list grows too big.